### PR TITLE
[Documentation] Improve documentation of loki tests

### PR DESCRIPTION
**Pull request by VilleMiekkoja**
## Description

Improves documentation related to loki testing

### CODE REVIEW

File: packages/react/DEVELOPMENT.md [hunk 1]
```diff
@@ -54,19 +58,27 @@ Also note the section below about visual regression tests and adding new compone
 
 ## Visual regression tests
 
-`hds-react` uses [Loki](https://loki.js.org/) for visual regression testing.
+`hds-react` uses [Loki](https://loki.js.org/) for visual regression testing. Make sure [git lfs](https://git-lfs.github.com/) is installed properly
+before running visual test or modifying reference images. Reference images are stored in this repository with git-lfs.
```
Comment (minevala) [inline packages/react/DEVELOPMENT.md:62]: before running visual test**s**?

**Review by minevala — APPROVED** [review]
👍

File: packages/react/DEVELOPMENT.md [hunk 2]
```diff
@@ -54,19 +58,27 @@ Also note the section below about visual regression tests and adding new compone
...
+
+### Running visual tests
 
 To run the visual regression tests, you must first build the storybook by running `yarn build-storybook`. Then you can proceed with the test run by running `yarn visual-test`.
 
-When adding a new component or after making visual changes to some existing component, you must update the corresponding the reference image. This is done by the following command:
+### Updating reference images
 
+When adding a new component or after making visual changes to some existing component, you must update the corresponding reference image.
+Before you can do this, you must start local react storybook by issuing the following command in the root of the whole project:
```
Comment (minevala) [inline packages/react/DEVELOPMENT.md:72]: start **the** local react

Comment (VilleMiekkoja) [reply packages/react/DEVELOPMENT.md]: Fixed :)

File: packages/react/DEVELOPMENT.md (hunk 1 above)
Comment (VilleMiekkoja) [reply packages/react/DEVELOPMENT.md]: Fixed :)

**Pull request merged**